cilium-dbg/cmd/statedb.go

```diff
@@ -27,7 +34,66 @@ var statedbDumpCmd = &cobra.Command{
 	},
 }
 
+func statedbTableCommand[Obj statedb.TabWritable](tableName string) *cobra.Command {
```

Comment:
nit: TabWritable to perhaps something like TableWriter?

Reply:
`TableWriter` to me is the thing that writes these `TabWritable` objects. I'll change it to `TableWritable` though.